setup.sh

```diff
@@ -111,15 +112,18 @@ echo "Select configuration mode:"
 echo ""
 echo "Current: ${config}"
 echo ""
-echo " 1 - Local Access (Powerwall 1, 2, or + using the Tesla Gateway on LAN) - Default"
-echo " 2 - Tesla Cloud  (Solar-only systems or Powerwalls without LAN access)"
+echo " 1 - Local Access   (Powerwall 1, 2, or + using the Tesla Gateway on LAN) - Default"
+echo " 2 - Tesla Cloud    (Solar-only systems or Powerwalls without LAN access)"
+echo " 3 - FleetAPI Cloud (Powerwall systems using Official Telsa API)"
```

Comment:
SETUP SCRIPT: This release adds the ability to use FleetAPI. If selected, it will kick off the FleetAPI setup process. If FleetpAPI is set up, the pypowerwall will always prefer that over cloud (Tesla Owners unofficial API) as FleetAPI is a Tesla provided official API.